src/EnergyPlus/SimulationManager.cc

```diff
@@ -1951,6 +1951,7 @@ namespace SimulationManager {
 					ShowWarningError( "Node Connection errors shown during \"fatal error\" processing may be false because not all inputs may have been retrieved." );
 					WarningOut = false;
 				}
+				if (CompSets(Count).CType == "COILSYSTEM:INTEGRATEDHEATPUMP:AIRSOURCE") continue;
```

Comment:
More specialization.  Feels like square peg in round hole.  Should we be restructuring at a different level to avoid these?  What is it about these units that is causing you to simply skip this error handling?